Tests/UnitTests/Networking/HTTPClientTests.swift

```diff
@@ -490,6 +490,46 @@ class HTTPClientTests: TestCase {
         expect(result?.value?.statusCode) == .success
     }
 
+    func testCachedRequestsIncludeETagHeader() {
```

Comment:
Turns out we didn't have tests for this!

Reply:
oops